src/proxy.js

```diff
@@ -131,6 +131,10 @@ function get(state, prop) {
 		drafts = state.copy
 	}
 
+	if (!Object.keys(state.base).includes(prop)) {
```

Comment:
`Object.getOwnPropertyDescriptor(state.copy || state.base, prop)` always returns `undefined` for getters and methods. So, to fix the problem, it's enough to check where the property exist or not in the state.